docs/Whitepaper/TCRs-for-Filtering.md

```diff
@@ -12,7 +12,7 @@ In general lines, token holders can help curate the list by 1) submitting videos
 
 Users are free to submit a video to the registry, along with a deposit stake of PTI. Proposals can be challenged or not, by those willing to 1) keep the list quality high, 2) try to earn a share of the applicant stake, 3) both. Challenges can be made to any new applicant, and also to already listed videos, by any token holder willing to match a MIN_DEPOSIT stake of at least equal value to the amount in stake for that video, at any time. If an application is not challenged, the stake becomes effectively bonded in the candidate’s listing, and can be undone at any time. If challenged, the stake is matched by challenger(s), and a voting round starts, with one vote per token granted for all token holders. After a threshold is crossed, the outcome is determined. 
 
```

Comment:
So is the video removed from the list a) at the moment it is challenged or only b) after it looses a challenge.

I'd vote for one (remove offensive content quickly), we have (some) spam protection because the challenger will loose its counterstake

Reply:
Well pointed. The original design emphasises listing happens only after APPLY_STAGE_LEN is over and no challenge was made + put into effect. But this is an obvious UX burden in our case, so we're already being permissive and making listing happen in first place. As for your question specifically, I think the whole idea revolves around having different actors challenge (1) and judge (vote - 2), with separate incentives so both can generate a representative outcome. I think it's saner to think, instead, of tweaking the "VOTE_QUORUM" in order to make it responsive to time (increases as time passes by), so that, in the beginning, a bit of effort is enough to tackle down a video that has to be removed quickly, and the overall structure is preserved. We must also note that the basic incentive for people not to do this (the stake that will be lost, sooner or later) is already there.